### PR TITLE
Add cmap argument for mne.viz.utils.plot_sensors

### PR DESCRIPTION
**Pull request by Genuster**
#### What does this implement/fix?
A `cmap` argument for the `mne.viz.utils.plot_sensors()`.

#### Additional information
The colormap currently used is mpl's 'jet'. `_get_cmap()` converts None to `rcParams["image.cmap"]`, therefore, this PR changes the default colormap of the function.

### CODE REVIEW

**Review by agramfort — APPROVED** [review]
Thanks @Genuster can you add an entry in latest.inc to document the improvement ? Thanks again

**Review by agramfort — COMMENTED** [review]
thx @Genuster 

can you see why CIs complain? thanks

**Comment by Genuster** [comment]
@agramfort, I tried to build the docs and check styling, but other than that I'm quite lost with CIs, sorry.

File: mne/viz/utils.py
Comment (larsoner) [inline mne/viz/utils.py:1020]: We've started adding these with any new `kwargs`, so now is a good time to do it
```suggestion
    *,
    cmap=None,
```

File: mne/viz/utils.py
Comment (larsoner) [inline mne/viz/utils.py:1080]: Code mode will probably look better here
```suggestion
        is list of list. If None, set to ``matplotlib.rcParams["image.cmap"]``.
```

**Review by larsoner — APPROVED** [review]
Other than two minor suggestions LGTM. I'll go ahead and push them so that all CIs run (they usually don't for new contributors) but if you don't like them @Genuster let me know and we can discuss and possibly revert!

**Comment by larsoner** [comment]
Will need https://github.com/mne-tools/mne-python/pull/11723 to be green, will merge that into this PR once it lands in `main`...

**Pull request merged**
